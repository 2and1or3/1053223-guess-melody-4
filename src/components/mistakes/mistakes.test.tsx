import * as React from "react";
import * as renderer from "react-test-renderer";

import Mistakes from './mistakes';


it(`Render Mistakes component`, () => {
  const tree = renderer
    .create(<Mistakes maxMistakes = {3} mistakes = {1} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
