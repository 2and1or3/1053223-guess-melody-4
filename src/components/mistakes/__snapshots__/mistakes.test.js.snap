// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Mistakes component 1`] = `
<div
  className="game__mistakes"
>
  <div
    className="wrong"
  />
  <div
    className=""
  />
  <div
    className=""
  />
</div>
`;
